donations/models.py

```
@@ -176,112 +199,51 @@ class Meta:
+    def to_dict(self):
+        return model_to_dict(self)
 
     async def fetch_usd_prices_async(self):
         fetch_prices = sync_to_async(self.fetch_usd_prices)
         await fetch_prices()
 
     ### Fetches USD prices for the Donation record and saves USD totals
     def fetch_usd_prices(self):
+        # TODO: remove duplicate logic with PotPayout.fetch_usd_prices
         # get existing values for stats adjustments later
         existing_total_amount_usd = self.total_amount_usd
```

Ruff (0.14.5)

Local variable `existing_total_amount_usd` is assigned to but never used

Remove assignment to unused variable `existing_total_amount_usd`

(F841)